ksqldb-common/src/main/java/io/confluent/ksql/parser/OutputRefinement.java

```diff
@@ -31,13 +31,22 @@ public enum OutputRefinement {
   CHANGES,
 
   /**
-   * Only final results should be materialized.
+   * Only final results should be materialized, using suppress(), ie, in-memory store.
    *
    * <p>For a table, this would mean output only the finalized result per-key.
    *
    * <p>For a stream, all events are final, so all are output.
    */
-  FINAL;
+  FINAL,
```

Comment:
Is this old value still needed after this PR? It should be obsolete, right? Existing queries have suppress nodes in their plans, and new queries will only ever set `FINAL_PERSISTENT` instead.

Reply:
🤔 -- fair point. Could be that we don't need it...

Wondering if we should remove it? Or if we should re-purpose it?

Reply:
The distinction would only be internal, right? As in, users will never see `FINAL` vs `FINAL_PERSISTENT`, and we also don't write anything of the sort into the command topic. If so, I think either's fine since we'd always be able to change it in the future. My vote is to repurpose `FINAL` if you feel confident that it's safe, else replace `FINAL` with `FINAL_PERSISTENT` since we know for sure that's safe.

Reply:
Yes, internal only, there is just `EMIT FINAL` syntax for the user and if this is merged it should always use `FINAL_PERSISTENT` internally.

> My vote is to repurpose

Ack. Will look into it. Was not 100% sure if it would be save, but seems it should be. Better to cleanup right away.